### PR TITLE
add header component + restyling changes

### PR DESCRIPTION
**Pull request by Toastjpg**

### CODE REVIEW

**Comment by Toastjpg** [comment]
TODO: this assumes that the userid is in the path params 

This will change after auth is fully integrated...

File: frontend/src/components/CreateMemo/page.tsx [hunk 1]
```diff
@@ -153,26 +157,38 @@ const CreateMemo = ({ }) => {
...
-        </span>
+        <h2 className="text-3xl font-bold mb-6 mt-6 text-blue-800">
+          Creating New Memo
+        </h2>
         <div className="map-container">
-          <label htmlFor='location' className="text-xl text-blue-800"> Location</label>
-          <link rel="stylesheet" href="https://unpkg.com/leaflet/dist/leaflet.css" />
-          <MapForm selectedLocation={selectedLocation} onMapClick={handleMapClick} />
+          <label htmlFor="location" className="text-xl text-blue-800">
+            {" "}
```
Comment (jerrydngzh) [inline frontend/src/components/CreateMemo/page.tsx:169]: what is {" "}?

**Review by jerrydngzh — APPROVED** [review]
yeah lgtm, i trust it works 🤠

File: frontend/src/components/CreateMemo/page.tsx (hunk 1 above)
Comment (Toastjpg) [reply frontend/src/components/CreateMemo/page.tsx:169]: no clue asking prettier

**Review by jerrydngzh — APPROVED** [review]
👍

**Pull request merged**